saleor/checkout/utils.py

```diff
@@ -197,13 +197,15 @@ def add_variants_to_checkout(
     to_reserve = to_create + to_update
     if reservation_length and to_reserve:
         updated_lines_ids = [line.pk for line in to_reserve + to_delete]
+        to_update_reserved_until = []
```

Comment:
Maybe a better name will be `lines_to_update_reservation_time`? What do you think, for me, it seems more user-friendly.